### ISSUE TITLE
Very old release with > version number is seen as a new release

### ISSUE THREAD

**Issue by rwmjones** [fedora-infra/anitya#1168]
Thread:

https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/UVAQP4DEOO7PAIKWMFORV4MSZDFNJIFU/

ocamlbuild (https://release-monitoring.org/project/12387/) comes from this repo: 

https://github.com/ocaml/ocamlbuild

Very old releases (eg. 4.02.3) have a higher version number from the current version (0.14.0) because this project was originally forked off from a different project.  I think Anitya should ignore these 5+ year old versions.

**Comment by Zlopez** [comment]
For Github, only last 50 tags are retrieved, so in this case it depends how much tags were created. If you want to ignore them, just add them to ignored versions in project settings and let me know, so I can delete them afterwards.

**Comment by pombredanne** [comment]
Would this be a poster child for epoch? e.g., 4.02.3 is epoch 0 and 0.14.0 is epoch 1?

**Comment by Zlopez** [comment]
The epoch idea doesn't sound bad, it could solve plenty of issues on various projects, but it will not be easy to implement.

**Comment by rwmjones** [comment]
No, after the upstream split happens, a new ocaml-ocamlbuild package was added to Fedora, so no epoch was needed as it was a completely new package.
